tutor/templates/build/openedx/Dockerfile

```diff
@@ -160,8 +160,10 @@ COPY --chown=app:app revisions.yml /openedx/config/
 ENV LMS_CFG /openedx/config/lms.env.yml
 ENV CMS_CFG /openedx/config/cms.env.yml
 ENV REVISION_CFG /openedx/config/revisions.yml
+ENV UWSGI_WORKERS=2
 COPY --chown=app:app settings/lms/*.py ./lms/envs/tutor/
 COPY --chown=app:app settings/cms/*.py ./cms/envs/tutor/
+COPY --chown=app:app settings/uwsgi.ini .
```

Comment:
These ENV and COPY statements should be in the "final" layer. Otherwise any change in the uwsgi.ini file will trigger useless rebuilds of the "production" layer.